tasks/docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Check current docker-compose version.
+  command: "{{ docker_compose_path }} --version"
+  register: docker_compose_vsn
+  check_mode: false
+  changed_when: false
+  failed_when: false
+
+- set_fact:
+    docker_compose_current_version: "{{ docker_compose_vsn.stdout | regex_search('(\\d+(\\.\\d+)+)') }}"
+  when: docker_compose_vsn.stdout is defined
+
+- name: Delete existing docker-compose version if it's different.
+  file:
+    path: "{{ docker_compose_path }}"
+    state: absent
+  when: >
+    docker_compose_current_version is defined
+    and (docker_compose_version | regex_replace('v', '')) not in docker_compose_current_version
+- name: Install Docker Compose (if configured).
```

Comment:
Fails for `update` operation.

![image](https://user-images.githubusercontent.com/58501088/190401963-5a189274-7135-443e-8a59-e4deedebe1ea.png)

Reply:
Giving sudo privilege resolved it.

Reply:
Added the required sudo permission statement